src/diagram.ts

```diff
@@ -170,6 +170,8 @@ export class Diagram {
             const linkLabelLayer = this.svg.append('g').attr('id', 'link-labels');
             const tooltipLayer = this.svg.append('g').attr('id', 'tooltips');
 
+            d3.select("body").on('keydown', this.undisplayNodes);
```

Comment:
Is there any way to restrict the event target? `body` looks too wide because some users embed inet-henge in their app, where they have `<form />` for example, typing in `<form />` will unexpectedly trigger `.undisplayNodes()` in that case.

Reply:
Umm, I'm afraid it's hard unfortunately 🤔 

```diff
@@ -241,6 +242,7 @@ export class Diagram {
     }

     undisplayNodes(): void {
+        console.log(event.target);  // This line always prints "<body />"
         // reset undisplayed nodes when escape key pressed
         if ((event as KeyboardEvent).keyCode === 27) {
             const node = d3.selectAll('.node')
```

Reply:
Unless nodes are selected, they don't disappear unexpectedly, so I'm not sure that would be a huge problem, but yes, I agree that having some plugin or something would be greater to prevent it from unexpected behaviors.